com.unity.formats.fbx/Editor/Unity.Formats.Fbx.Editor.api

```diff
@@ -0,0 +1,125 @@
+// This file is generated. Do not modify by hand.
+// XML documentation file not found. To check if public methods have XML comments,
+// make sure the XML doc file is present and located next to the scraped dll
+namespace UnityEditor.Formats.Fbx.Exporter
+{
+    public static class ConvertToNestedPrefab
+    {
+        public static UnityEngine.GameObject Convert(UnityEngine.GameObject toConvert, string fbxDirectoryFullPath = default(string), string fbxFullPath = default(string), string prefabDirectoryFullPath = default(string), string prefabFullPath = default(string), UnityEditor.Formats.Fbx.Exporter.ConvertToPrefabSettingsSerialize exportOptions = default(UnityEditor.Formats.Fbx.Exporter.ConvertToPrefabSettingsSerialize));
```

Comment:
Export doesn't change the scene (or if it does there's a bug we should fix someday).

Convert does change the scene. We're trying to emulate the semantics that users see when they drag a gameobject from the hierarchy to the project folder. In that case, what you see in the scene is that the gameobject you dragged becomes an instance of the new prefab, and all references in the scene to anything in that hierarchy stays connected to it.

Reply:
>If you set this paths to things outside of "Assets" folder does it work ? (I see those paths are sent to prefab functions)

it will throw an exception in this case, I have added unit tests for it.